src/output-filestore.c

```diff
@@ -403,7 +403,8 @@ static OutputInitResult OutputFilestoreLogInitCtx(ConfNode *conf)
 
     intmax_t version = 0;
     if (!ConfGetChildValueInt(conf, "version", &version) || version < 2) {
-        result.ok = true;
+        SCLogWarning(SC_WARN_DEPRECATED,
+            "File-store v1 been removed. Please update to file-store v2.");
```

Comment:
Does this lead to failure with `-T`?

Reply:
Yes, and --init-errors-fatal. Otherewise its just a warning and Suricata continues to startup.